elasticdl/python/common/model_handler.py

```diff
@@ -221,20 +220,85 @@ def _clone_function(layer):
         )
 
     @staticmethod
-    def _replace_embedding_attributes_for_subclass(
-        model, src_embedding_class, dst_embedding_class
-    ):
-        """Replace the keras embedding attribute with
-        elasticdl.layers.Embedding layer.
+    def _clone_model_with_keras_embedding(model):
+        """Clone a new model and replace the `elasticdl.layers.Embedding`
+        layers with `tf.keras.layers.Embedding` or `SparseEmbedding` layers
+        """
+
+        def _clone_function(layer):
+            if type(layer) == Embedding:
+                # ElasticDL embedding only accept a string type initializer
+                if layer.combiner is not None:
```

Comment:
Add comments here:
Only for the scenario of SparseEmbedding, the combiner is not None